### PR TITLE
tweaks to PR #1860: tweak text describing per minute system limit to avoid term "user"

### PR DESCRIPTION
**Pull request by csantanapr**
@sjfink OK with additional tweaks?

superseed #1860

### CODE REVIEW

File: docs/reference.md [hunk 1]
```diff
@@ -482,7 +482,7 @@ Triggers are subject to a firing rate per minute as documented in the table belo
 
 | limit | description | configurable | unit | default |
 | ----- | ----------- | ------------ | -----| ------- |
-| minuteRate | a user cannot fire more than this many triggers per minute | per user | number | 60 |
+| minuteRate | no more than N triggers may be fire per namespace per minute | per user | number | 60 |
```
Comment (rabbah) [inline docs/reference.md]: may be fired ...

Comment (csantanapr) [inline docs/reference.md]: thanks editor-in-chief 😹

**Comment by csantanapr** [comment]
@rabbah approve review now with 9970548ae9d51e3c58ea2a2a56af0f1bfba79185 ?

**Comment by csantanapr** [comment]
merge 87f8673526dd1ff0faaa9fe4e4060153df8ff708

**Pull request closed without merging**